### ISSUE TITLE
Mail settings in Back End section

### ISSUE THREAD

**Issue by bflanagin** [LeafletBomb/leafletbomb#33]
We need a way to change some of the default settings for the mail message. Right now its hard coded which is never the way you want to do things.

**Comment by bflanagin** [comment]
Added the most obvious one. Need to add less obvious ones when I come up with them

**Comment by ryanleesipes** [comment]
We need to do something to ensure that folks know there has to be something in that field in order for it to work (at least with SendGrid).

**Comment by ryanleesipes** [comment]
Are you planning on putting all the settings in the backend section? (the ones in the settings right now)?

**Comment by ryanleesipes** [comment]
#49 related.

**Comment by bflanagin** [comment]
All server side back end options, Email, Scheduler, maybe a few other things I haven't thought of yet.

**Comment by bflanagin** [comment]
Will need to add more options to the settings area but this is no longer a bug but more of a feature "incomplete"

**Comment by ryanleesipes** [comment]
I've been thinking about this. I think Email settings (like server and auth maybe should be outside the app anyway). I think a system administrator will fill that stuff there, and may not care to have the PR folks know.

**Comment by bflanagin** [comment]
Agreed, and we can't put them inside the meteor app anyway due to some of the other issues I've had. I was thinking more general options. Like the ones already there and a host of other options like what goes IN the email. Later we can add a template selector for the email as well.
